### ISSUE TITLE
"Converting chance" does not do anything for Boss's

### ISSUE THREAD

**Issue by LeehamElectronics** [kangarko/Boss#635]
Hello! I have "converting" enabled in my config, however when I spawn creepers with a normal vanilla egg, the creepers are not converting into my custom creeper even at 100% converting chance. The only time they convert into the custom creeper is when I set the 'World spawning' chance to a value above 0, however that enables random spawning as well, and it makes the 'converting' chance ambiguous because it makes no difference.

I think implementing the following will fix this:

- Have 2 different buttons in the GUI inside the spawning menu, by default they are _'red stone blocks' = 'off',_ when the player clicks it they turn into _'emerald blocks' meaning 'on'_

- One toggle switch ENABLES _'random spawning'_, which can further be adjusted in the world spawning menu as it is. The other toggle switch ENABLES _'converting'_ which can be configured separately, so instead of just having a 'Spawning in Worlds' menu, we will have a 'Converting in Worlds / biomes' menu as well.

**In summary,** to fix this issue we really need two toggle switches in the GUI to enable and disable Converting / Spawning, and we also need a separate sub-menu for Converting in worlds / biomes, not just 'Spawning in worlds / biomes'
It would also be nice to simply write in the chat what percentage of spawn chance per world rather than clicking 50 times to get to 100%

Thanks, I know its a lot of writing, but I feel like this is really important, I am trying to setup a MMO HardCore Survival server and converting and spawning custom mobs is really important for me.

**Comment by kangarko** [comment]
Please see my other comment regarding new spawning system and let me know your thoughts. Current system cannot be modified this way, internally.
